src/app/modules/shared/components/dialog/dialog.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="modal-dialog" role="document">
+  <div class="modal-content" *ngIf="list">
+    <div class="modal-header">
+      <h5 class="modal-title" id="exampleModalLabel">{{ title }}</h5>
+      <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </div>
+    <div class="modal-body">
+      Are you sure you want to delete <b>{{ list.project_name || list.name }}</b>
```

Comment:
Don't we need a question mark at the end of this sentence?

Reply:
We do need it for sure. So messages should look like:

Are you sure you want to delete **XYZ**?

Reply:
yes this is working, I need to put this {{ list.project_name || list.name }} because the objects don't have the same names.